src/components/RenderHtml/RenderHtml.tsx

```diff
@@ -95,7 +96,13 @@ export function RenderHtml({ html, withMentions = false, ...props }: Props) {
             const dataType = attribs['data-type'];
             const isMention = dataType === 'mention';
             const style = attribs['style'];
-            const hexColor = style?.match(/color:#([0-9a-f]{6})/)?.[1];
+            let hexColor = style?.match(/color:#([0-9a-f]{6})/)?.[1];
+            const [, r, g, b] = style?.match(/color:rgba?\((\d+), (\d+), (\d+),? ?(\d+)?\)/) ?? [];
+            const rgbColors = [r, g, b].filter(isDefined).map((color) => parseInt(color, 10));
```

Comment:
what do you think about .toString(16) here too?

Reply:
I guess it's possible, let me give it a try